src/listener.rs

```diff
@@ -20,6 +21,21 @@ impl std::fmt::Debug for UnixSeqpacketListener {
 	}
 }
 
+impl AsFd for UnixSeqpacketListener {
+	fn as_fd(&self) -> std::os::unix::prelude::BorrowedFd<'_> {
+		self.io.get_ref().as_fd()
+	}
+}
+
+impl TryFrom<OwnedFd> for UnixSeqpacketListener {
+	type Error = std::io::Error;
+
+	fn try_from(fd: OwnedFd) -> Result<Self, Self::Error> {
+		// # Safety: we solely own the FD
+		unsafe { Self::from_raw_fd(fd.into_raw_fd()) }
```

Comment:
```suggestion
		Self::new(FileDesc::new(fd))
```

No need for unsafe here :)